slash/react/src/Accordion/Client/Accordion.tsx

```diff
@@ -1,8 +1,103 @@
-import "@axa-fr/design-system-slash-css/dist/Accordion/Accordion.client.scss";
-import { AccordionItem } from "./AccordionItem";
+import type React from "react";
+import { useCallback, useMemo, type ComponentProps } from "react";
+import { Tag } from "../../Tag";
+import {
+  BREAKPOINT,
+  getComponentClassName,
+  useIsSmallScreen,
+} from "../../utilities";
+import { AccordionTagDateContainer } from "./AccordionTagDateContainer";
 
-export const Accordion = ({ children }: React.PropsWithChildren) => {
-  return <div className="af-accordion-client">{children}</div>;
+type AccordionProps = {
+  title: string;
```

Comment:
on peut pas mettre ReactNode ?

Reply:
Resolved in: https://github.com/AxaFrance/design-system/pull/459/commits/d0897a888c591870d2077812b921dfc51f404246